modules/libvideo/g_video.c

```diff
@@ -275,6 +275,12 @@ extern int bennugd_content_height;
     depth=libretro_depth;
     bennugd_content_width = width;
     bennugd_content_height = height;
+
+#ifdef __WEBOS__
+    // Force SDL video driver to LIBRETROvideo when running on webOS
+    setenv("SDL_VIDEODRIVER", "LIBRETROvideo", 1);

```

Comment:
Ah, I see. The root cause is that on webos SDL_VIDEODRIVER is set. But I would fix it in a different way. There is an internal SDL 1.2 inside BennuGD_libretro that is heavily modified to redirect everything to the frontend. There is only a libretro driver left. I deleted all other. It doesn't make any sense that the internal implementation checks any environment variables. It is also a bit dangerous to modify them. There might be an sdl library outside of the core that this variable potentially influences. The video driver of the external SDL is probably alsready initialized at this point so it does no harm right now. But the core should leave environment varaibles alone.